felix/bpf-gpl/parsing.h

```diff
@@ -175,14 +175,13 @@ static CALI_BPF_INLINE int tc_state_fill_from_nexthdr(struct cali_tc_ctx *ctx, b
 		break;
 #ifdef IPVER6
 	case IPPROTO_ICMPV6:
-		/* XXX for now, just allow */
-		CALI_DEBUG("ICMPv6: allow");
-		goto allow;
+		ctx->state->sport = ctx->state->dport = 0;
+		CALI_DEBUG("ICMP; type=%d code=%d\n",
+				icmp_hdr(ctx)->type, icmp_hdr(ctx)->code);
+		break;
 #else
 	case IPPROTO_ICMP:
-		ctx->state->icmp_type = icmp_hdr(ctx)->type;
```

Comment:
Do we not use this for policy?  I thought our policy model supported matching on it (so it would need to be in the state for the policy program to see).

Reply:
hmmm right , policy ... :thinking:

Reply:
I made this change because verifier was exploding and stack was overflowing. I think I found the right permutation where everybody is happy.